.github/workflows/ubuntu.yml

```diff
@@ -26,7 +26,7 @@ jobs:
             software-properties-common gcc ruby ruby-dev libffi-dev make libxml2-dev libxslt1-dev
 
           # Install fonts
-          sudo apt-get -y install fonts-freefont-ttf fonts-droid-fallback
+          sudo apt-get -y install fonts-freefont-ttf
```

Comment:
Remove Freefont?

Reply:
Yes, you're right. It should be removed. I've fixed it.